src/search/FindReplace.js

```diff
@@ -34,31 +34,38 @@
 define(function (require, exports, module) {
     "use strict";
 
-    var CommandManager      = require("command/CommandManager"),
-        Commands            = require("command/Commands"),
-        MainViewManager     = require("view/MainViewManager"),
-        Strings             = require("strings"),
-        StringUtils         = require("utils/StringUtils"),
-        Editor              = require("editor/Editor"),
-        EditorManager       = require("editor/EditorManager"),
-        FindBar             = require("search/FindBar").FindBar,
-        FindUtils           = require("search/FindUtils"),
-        FindInFilesUI       = require("search/FindInFilesUI"),
-        ScrollTrackMarkers  = require("search/ScrollTrackMarkers"),
-        _                   = require("thirdparty/lodash"),
-        CodeMirror          = require("thirdparty/CodeMirror/lib/codemirror");
+    var CommandManager       = require("command/CommandManager"),
+        Commands             = require("command/Commands"),
+        MainViewManager      = require("view/MainViewManager"),
+        Strings              = require("strings"),
+        StringUtils          = require("utils/StringUtils"),
+        Editor               = require("editor/Editor"),
+        EditorManager        = require("editor/EditorManager"),
+        FindBar              = require("search/FindBar").FindBar,
+        FindUtils            = require("search/FindUtils"),
+        FindInFilesUI        = require("search/FindInFilesUI"),
+        ScrollTrackMarkers   = require("search/ScrollTrackMarkers"),
+        BracketsSearchCursor = require("search/BracketsSearchCursor"),
+        _                    = require("thirdparty/lodash"),
+        CodeMirror           = require("thirdparty/CodeMirror/lib/codemirror");
 
     /**
      * Maximum file size to search within (in chars)
      * @const {number}
      */
-    var FIND_MAX_FILE_SIZE  = 500000;
+    var FIND_MAX_FILE_SIZE  = 16000000;
```

Comment:
>15MB does seem as too high to me (given Brackets is an editor for HTML5 files). Is it just because the size limitation is no longer particularly limiting performance? Or it's taken from some real-life scenarios?